app/api/vendor/route.ts

```
@@ -92,28 +127,37 @@ export async function POST(request: Request) {
+      requestData.serviceCategory as ServiceCategoryType;
+
+    // --- Generate Unique Slug ---
+    const uniqueSlug = await generateUniqueSlug(requestData.name);
+    console.info("Generated unique slug:", uniqueSlug);
 
-    // Create vendor record
+    // Create vendor record including the slug
     const newVendor = await prisma.vendor.create({
       data: {
         name: requestData.name,
+        slug: uniqueSlug, // Save the generated slug
```

Biome (1.9.4)

[error] This property value named slug is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named slug


(lint/suspicious/noDuplicateObjectKeys)